fhd_output/fhd_quickview.pro

```diff
@@ -276,13 +276,14 @@ ENDFOR
 IF Keyword_Set(cal_source_flag) THEN cal_sources = source_image_generate(cal_source_arr_out,obs_out,pol_i=4,resolution=16,$
     dimension=dimension,restored_beam_width=restored_beam_width,_Extra=extra)
 
-; renormalize based on weights
-renorm_factor = get_image_renormalization(obs_out,weights_arr=weights_arr,beam_base=beam_base_out,filter_arr=filter_arr,$
-  image_filter_fn=image_filter_fn,pad_uv_image=pad_uv_image,degpix=degpix,/antialias)
-for pol_i=0,n_pol-1 do begin
-  *instr_dirty_arr[pol_i]*=renorm_factor[pol_i]
-  IF model_flag THEN *instr_model_arr[pol_i]*=renorm_factor[pol_i]
-endfor
+IF keyword_set(image_filter_fn) THEN IF image_filter_fn NE 'filter_uv_weighted' THEN BEGIN ; renormalize based on weights
```

Comment:
We need to set the default for `image_filter_fn` to `filter_uv_uniform` at the start of this file. Otherwise it just gets set to that inside `dirty_image_generate` and that could be surprising. That means that you can treat it as guaranteed to be defined and can skip the `IF keyword_set(image_filter_fn)`... test here.

Then, you should modify `get_image_renormalization` so that it returns the correct normalization regardless of the filter function. For example, it should know to return `1` for `filter_uv_weighted` (right at the start!), and should divide by `(degpix*!DtoR)^2. in all other cases. I think other code relies on `get_image_renormalization`, so you should add a new keyword to convert to Jy/sr and set that when you call it here.

Reply:
After you've made the above changes, you shouldn't need an `IF...` statement here at all.

Reply:
I also think this should be resolved now but could use another review